File: Dynamic Programming/Fibonacci Series/FibonacciWithoutDPCode/FibonacciWithoutDP.ts
class Fib{
    getNthFibonacciNumber = (n: number): number => {
        if(n == 0 || n == 1){
            return n;
        }
        
        return this.getNthFibonacciNumber(n-1) + this.getNthFibonacciNumber(n-2);
    }
}

File: Dynamic Programming/Fibonacci Series/FibonacciWithoutDPCode/FibonacciWithoutDP.java
public class Fib
{
    public static int getNthFibonacciNumber(int n)
    {
        if(n == 1 || n == 0)
        {
            return n;
        }

        return getNthFibonacciNumber(n-1) + getNthFibonacciNumber(n-2);
    }

    public static void main(String[] args)
    {
        int n = 8;
        System.out.println(getNthFibonacciNumber(n));
    }
}